bin/run-addon

```diff
@@ -115,7 +115,7 @@ if [[ -z "$no_auto" ]] ; then
 fi
 
 cd build/addon
-jpm run --binary $binary --profile ../../Profile --no-copy \
+jpm run --binary $binary --profile ../../dist/Profile --no-copy \
```

Comment:
It is vestigal, yes.  I moved it to ./Profile in the root of the project (no particular reason for that location, but I didn't want it in build/ because then make clean would wipe it)